README-dist-ops.md

```diff
@@ -90,24 +90,24 @@ dftool calc $CSV_PATH 10000 OCEAN
 
 ### Step 4: Run dispense
 
-Get a working account. We call it dftool_acct. Either use a previous one, or create a new one. For the latter::
+Get a working account. We call it DFTOOL_ACCT. Either use a previous one, or create a new one. For the latter::
```

Comment:
Please move all changes wrt `dftool_acct` to a new PR. (With an appropriate github issue attached.)

Why: each PR should be well-defined and tightly scoped. Way easier to review and approve. Way easier to maintain. To be specific here: small refactorings, especially ones that have nothing to do with the main problem the PR is trying to solve, should not be in the PR.

Reply:
I misread this, it looks good at is.